integration_tests/pom.xml

```diff
@@ -59,38 +59,40 @@
             <scope>provided</scope>
         </dependency>
         <dependency>
-            <groupId>com.nvidia</groupId>
-            <artifactId>rapids-4-spark_${scala.binary.version}</artifactId>
-            <version>${project.version}</version>
-            <scope>provided</scope>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <scope>test</scope>
         </dependency>
-        <!-- use aggregator jar because accessing internal classes -->
         <dependency>
-            <groupId>com.nvidia</groupId>
-            <artifactId>rapids-4-spark-aggregator_${scala.binary.version}</artifactId>
-            <version>${project.version}</version>
-            <classifier>${spark.version.classifier}</classifier>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>com.nvidia</groupId>
-            <artifactId>rapids-4-spark-udf-examples_${scala.binary.version}</artifactId>
-            <version>${project.version}</version>
-            <scope>test</scope>
+            <groupId>org.apache.spark</groupId>
+            <artifactId>spark-sql_${scala.binary.version}</artifactId>
+            <version>${spark.test.version}</version>
+            <scope>compile</scope>
```

Comment:
The integration tests are still depending on the spark-sql artifact and it will appear in the classpath when the tests are run.

Reply:
I agree with **must not** depend on the **rapids-4-spark-sql** artifact, but we must depend on **spark-sql** to run the Scala test cases. When running the Python test cases the spark-sql artifact will appear in the classpath because of command "$SPARK_HOME/bin/pyspark". But for the Scala test cases is a different path, the class path is specified by "mvn test".
I'll move some unit tests to the rapids-4-spark-sql module, but the unit tests can also run in integration test.
I want to discuss if this solution is ok?